File: src/SolucionesJava/Solucion_2.java
package SolucionesJava;

import java.util.Scanner;

public class Solucion_2 {
    public static void solucion_2(){
        Scanner scanner = new Scanner(System.in);

        System.out.print("introduce un número: ");
        int numero = scanner.nextInt();

        if (numero % 2 == 0) {
            System.out.println("el numero " + numero + " es par.");
        } else {
            System.out.println("el numero " + numero + " es impar.");
        }
    }
}

File: src/SolucionesKotlin/Solucion_2.kt
package SolucionesKotlin

fun Solucion_2(){
    println("Introduce un numero")
    val numero = readln().toInt()
    parImpar(numero)
}

fun parImpar(num: Int){
    if (num%2 == 0){
        println("es par")
    }else println("es impar")
}

